deploy.sh

```
@@ -1,7 +1,43 @@
+	   --project=${PROJECT} \
+	   --image=${LOCATION}-docker.pkg.dev/${PROJECT}/artifacts/${SERVICE} \
+	   --base-image=${LOCATION}-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python312 \
+	   --region=${LOCATION} \
+           --no-allow-unauthenticated \
+	   --concurrency=1 \
+	   --max-instances=5 \
+	   --timeout=900 \
+	   --cpu=0.2 \
+	   --memory=256Mi \
+	   --cpu-boost \
+	   --update-env-vars=PROJECT_NUMBER=$PROJECT_NUMBER \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)